Libraries/IntegrationServiceLibrary.psm1

```diff
@@ -237,12 +237,12 @@ function Get-IPv4ViaKVP {
 						if ($addr) {
 							return $addr
 						}
-						$retryTime++
-						Start-Sleep -Seconds 10
 					}
 				}
 			}
 		}
+		$retryTime++
```

Comment:
retryTime is increasing 2 times per loop.

Reply:
Also see the entire work flow. If rawData is null, then there is a chance for 'exception'. This does not have try-catch-finally block.

Reply:
Please note the keyword 'continue' in line 213. The keyword resolved the 2 questions you mentioned.

Reply:
ok